order/admin.py

```diff
@@ -1,6 +1,12 @@
 from django.contrib import admin
-from order.models import Status
+from order.models import Status, Declaration
 
 admin.site.register(Status)
 
-# Register your models here.
+
+@admin.register(Declaration)
+class DeclarationAdmin(admin.ModelAdmin):
+
+    def save_model(self, request, obj, form, change):
+        obj.user = request.user
```

Comment:
Let admin be able to choose user as well. If no user is defined, then user should be THAT user